hw-04-weather-api-app/js/app.js

```diff
@@ -0,0 +1,75 @@
+
+/*
+Add a input box to enter city name apply a submit button on the page when user inputs city make sure a function calls openweather api.
+
+when city is entered display results should include city name, current temp, weather description, min temp,max temp.
+
+Once city is picked results should display but also change when adding an if statement to check if the weather is either above 85 degree or below 40.*/
+$(function () {
+  $('#search').click(() => {
+      event.preventDefault()
+      //console.log('button works')
+      const city = $('#city-type').val()
+
+        })
+          function seachCity(searchterm) {
+
+            // note added async to function
+
+            async function getWeather (searchTerm) {
```

Comment:
@Ashbk32 looks like you're attempting to declare this function inside of the `searchCity()` function. You generally want to avoid that because it complicates how the functions will get called.